### ISSUE TITLE
provide mask files

### ISSUE THREAD

**Issue by listar0810** [zhishenhuang/mri#1]
thx for providing codes,could u provide mask files especially codes in unet_train.py
`# Mar 14, try fixed Loupe mask
    maskfile = '/mnt/shared_a/checkpoints/leo/mri/loupe_mask_8fold.pt'
    fix_mask = torch.load(maskfile)['mask']`
could u provide "loupe_mask_8fold.pt“ mask files.
Many thx

**Comment by NorthernPeach** [comment]
Hello, Zhi Shen,
have the same request, 
could you provide the "loupe_mask_8fold.pt“ mask files, please.

**Comment by zhishenhuang** [comment]
The file `loupe_mask_8fold.pt` was generated by applying a trained loupe model on the data to obtain corresponding masks. I have provided the `loupe` model training files in this repo as well.

I do not have the exact file I used back in March this year at hand after I moved to a new institution.

**Comment by NorthernPeach** [comment]
> The file `loupe_mask_8fold.pt` was generated by applying a trained loupe model on the data to obtain corresponding masks. I have provided the `loupe` model training files in this repo as well.
> 
> I do not have the exact file I used back in March this year at hand after I moved to a new institution.

After I train LOUPE via loupe_train_v2.py, the "loupe_8fold_base_8_epoch_0.pt" files are obtained
I do after that "alternate_train_v3.py" with the random initialization for U-Net and MNet, however, seems like some part of the steps before running alternate training is missing. During the training process, there is no improvement for the generated masks but degradation. 
Could you, please, tell in what the correct order is for running your code?

**Comment by zhishenhuang** [comment]
I do not think you need `loupe_mask_8fold.pt` in the alternating training framework. That file in the `unet_train.py` seemed to be an attempt to train an Unet that can denoise images masked by provided 'fixed masks' from Loupe procedure. One can certainly comment out those two lines.

The alternating training framework functions by its own (i.e. the file `alternate_train_v3.py`). It does not necessarily depend on the output from another file. You might want to pre-process your dataset by normalising each images to a moderate numerical range such as [0,1]. I should have provided most details in the writeup.
